mslib/utils/mssautoplot.py

```diff
@@ -323,68 +428,125 @@ def draw(self):
             else:
                 print(str(e))
         else:
             print("Plot downloaded!")
-
-    for flight, section, vertical, filename, init_time, time in \
-        config["automated_plotting_flights"]:
+            return True
+        return False
+    if ctx.obj is not None:
+        pdlg.setValue(4)
+    flag = False
+    for flight, section, vertical, filename, init_time, time in config["automated_plotting_flights"]:
+        if ctx.obj is not None:
+            pdlg.setValue(8)
         for url, layer, style, elevation in config[sec]:
             if vtime == "" and stime == "":
                 no_of_plots = 1
-                draw(no_of_plots)
+                flag = draw(no_of_plots)
             elif intv == 0:
                 if itime != "":
-                    init_time = datetime.strptime(itime, "%Y-%m-%dT" "%H:%M:%S")
-                time = datetime.strptime(vtime, "%Y-%m-%dT" "%H:%M:%S")
+                    init_time = datetime.strptime(itime, "%Y-%m-%dT%H:%M:%S")
+                time = datetime.strptime(vtime, "%Y-%m-%dT%H:%M:%S")
                 if ftrack != "":
                     flight = ftrack
                 no_of_plots = 1
-                draw(no_of_plots)
+                flag = draw(no_of_plots)
             elif intv > 0:
                 if itime != "":
-                    init_time = datetime.strptime(itime, "%Y-%m-%dT" "%H:%M:%S")
-                starttime = datetime.strptime(stime, "%Y-%m-%dT" "%H:%M:%S")
-                endtime = datetime.strptime(etime, "%Y-%m-%dT" "%H:%M:%S")
+                    init_time = datetime.strptime(itime, "%Y-%m-%dT%H:%M:%S")
+                starttime = datetime.strptime(stime, "%Y-%m-%dT%H:%M:%S")
+                endtime = datetime.strptime(etime, "%Y-%m-%dT%H:%M:%S")
                 i = 1
                 time = starttime
                 while time <= endtime:
                     logging.debug(time)
                     if ftrack != "":
                         flight = ftrack
                     no_of_plots = i
-                    draw(no_of_plots)
+                    flag = draw(no_of_plots)
                     time = time + timedelta(hours=intv)
-                    i = i + 1
+                    i += 1
             else:
                 raise Exception("Invalid interval")
+    if ctx.obj is not None:
+        pdlg.setValue(10)
+        pdlg.close()
+        if flag:
+            QMessageBox.information(
+                ctx.obj,  # The parent widget (use `None` if no parent)
+                "SUCCESS",  # Title of the message box
+                "Plots downloaded successfully."  # Message text
+            )
+        else:
+            QMessageBox.information(
+                ctx.obj,  # The parent widget (use `None` if no parent)
+                "FAILURE",  # Title of the message box
+                "Plots couldnot be downloaded."  # Message text
+            )
 
 
 if __name__ == '__main__':
-    main()
+    cli_tool()
```

Comment:
this needs an additional change in https://github.com/Open-MSS/MSS/blob/develop/setup.py#L39

I am not convinced by the new name.